php-checks/src/main/java/org/sonar/php/checks/OpeningPHPTagCheck.java

```diff
@@ -60,13 +59,12 @@ public void visitNode(AstNode astNode) {
    * The node contains everything before the first opening include HTML if present
    * this allows to ensure reporting the issue on the correct line.
    */
-  private int getLineToReport(AstNode node) {
-    return node.getTokenValue().split("(?:\r)?\n|\r").length;
+  private int getLineToReport(String openingTag) {
+    return openingTag.split("(?:\r)?\n|\r").length;
   }
 
-  private boolean isNotAuthorisedTag(AstNode node) {
+  private boolean isNotAuthorisedTag(String openingTag) {
     int offset = 1 + LONG_TAG.length();
-    String openingTag = node.getTokenOriginalValue();
 
     if (openingTag.length() > offset) {
       openingTag = openingTag.substring(offset);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "openingTag". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)